libiop/protocols/encoded/lincheck/basic_lincheck_aux.tcc

```diff
@@ -94,24 +94,28 @@ std::shared_ptr<std::vector<FieldT>> multi_lincheck_virtual_oracle<FieldT>::eval
      * [TODO: cite Succinct Aurora] instead of powers of alpha. */
```

Comment:
Can you change that citation to be [BCGGRS19]?

Reply:
err, no the original citation was right. I meant can you change the TODO haha

Reply:
> err, no the original citation was right. I meant can you change the TODO haha

Oh my bad, yeah will be fixed in upcoming commit.